### PR TITLE
Dependencies update

### PR DESCRIPTION
**Pull request by chenrui333**
As the latest golang version is v1.13.1, it would be upgrade to golang v1.13 (which is latest and greatest).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 language: go
-sudo: required
```
Comment (tomaszsek) [inline .travis.yml]: sudo is required for run minikube - e2e

**Comment by tomaszsek** [comment]
@cheneui333 Please look at Travis build is failing now.

Cheers

File: .travis.yml (hunk 1 above)
Comment (chenrui333) [reply .travis.yml]: interesting, in Travis Linux build, the `sudo` permission is not quite required.

https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

**Comment by chenrui333** [comment]
@tomaszsek Should be good for the merge. Thanks!

**Comment by tomaszsek** [comment]
@chenrui333 Thanks for the PR!

**Pull request merged**